packages/devtools_app/lib/src/screens/memory/memory_heap_tree_view.dart

```diff
@@ -694,20 +695,21 @@ class HeapTreeViewState extends State<HeapTree>
 
           // Keep the bubble displayed for a few seconds.
           if (removeUpdateBubble != null) {
-            removeUpdateBubble.cancel();
+            removeUpdateBubble!.cancel();
```

Comment:
replace this block with `removeUpdateBubble?.cancel()`

Reply:
done